queries/ne.jinja2

```diff
@@ -32,7 +32,9 @@ SELECT
 {% endif %}
     'min_zoom', mz_boundary_min_zoom,
     'featurecla', featurecla,
-    'scalerank', scalerank
+    'scalerank', scalerank,
+    'ne_id', ne_id,
+    'BRK_A3', BRK_A3
```

Comment:
Tilezen's convention is lowercase only... so:

>     'BRK_A3', brk_a3

Reply:
fixed in [4624bff](https://github.com/tilezen/vector-datasource/pull/2079/commits/4624bff282298e346af4fb4f60dda5f1d1b86a7d)